repository/repository.go

```diff
@@ -423,3 +427,22 @@ To find more information on specific steps, please see documentation [here](http
 - [ ] QA: Final validation of above PR and tracked through the linked ticket
 - [ ] PJM: Close the milestone in GitHub.
 `
+
+func ContentByFileNameAndCommit(owner, repo, commitHash, filePath string, httpClient *http.Client) ([]byte, error) {
```

Comment:
The GitHub client provides Repositories.GetContents, which I think covers this functionality.

Reply:
Removed